mods/cnc/chrome/multiplayer-createserver.yaml

```diff
@@ -92,6 +92,15 @@ Container@MULTIPLAYER_CREATESERVER_PANEL:
 							Height: 20
 							Font: Regular
 							Text: Automatic port forwarding
+						Label@UPNP_NOTICE:
+							X: 130
+							Y: 180
+							Width: 200
+							Height: 25
+							Font: Tiny
+							Align: Left
+							WordWrap: True
+							Text: Requires Network Discovery (UPnP). Enable in Settings - Advanced.
```

Comment:
We only disable UPnP if there's an error, so i'm not sure we should be casually telling people how to reenable it.  That will cause confusion when the game immediately disables it again.